python/lsst/ip/isr/isrTask.py

```diff
@@ -1594,7 +1597,8 @@ def ensureExposure(self, inputExp, camera, detectorNum):
             # Assume this will be caught by the setup if it is a problem.
             return inputExp
         else:
-            raise TypeError(f"Input Exposure is not known type in isrTask.ensureExposure: {type(inputExp)}.")
+            raise TypeError("Input Exposure is not known type in isrTask.ensureExposure: %s." %
+                            (type(inputExp)))
```

Comment:
Either drop the brackets around `type(inputExp)` or add a comma, `(type(inputExp),)`: as it is, the brackets don't add anything.